cfme/common/host_views.py

```diff
@@ -82,6 +82,14 @@ class HostDetailsEntities(View):
     summary = ParametrizedView.nested(ParametrizedSummaryTable)
 
 
+class HostNetworkDetailsView(View):
+    network_tree = BootstrapNav(".//div[contains(@class,'treeview')]/ul")
+
+    @property
+    def is_displayed(self):
+        return self.network_tree.is_displayed
```

Comment:
**Question** Are there any other pages with this network tree? I think it is worth expanding this `is_displayed` a bit further. Possible checking the breadcrumb and title text

Reply:
I honestly don't have seen it anywhere else. But yes I will consider extending its is_displayed.